src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -103,9 +103,7 @@ class CourtController(
     )
     @RequestParam(name = "court", required = false)
     court: String?
-  ): List<VideoLinkBookingResponse> {
-    return courtService.getVideoLinkBookingsForPrisonAndDateAndCourt("WWI", date, court)
-  }
+  ) = courtService.getVideoLinkBookingsForPrisonAndDateAndCourt("WWI", date, court)
```

Comment:
reckon we can probably get rid of this endpoint now.. not as part of this PR but we could check to see if its still being used and get rid of it